test/docs/sotd/deprecated2015.html

```diff
@@ -8,10 +8,10 @@
   <body>
     <div class="head">
       <h1>Simple baseline headers valid document</h1>
-      <h2>W3C Working Draft 15 March 2017</h2>
+      <h2>W3C Working Draft 15 March 2018</h2>
     </div>
     <h2>Status of This Document</h2>
-    <p>This document is governed by the <a id="w3c_process_revision" href="https://www.w3.org/2015/Process-20150901/">1 September 2015 <abbr title="World Wide Web Consortium">W3C</abbr> Process Document</a>.</p>
+    <p>This document is governed by the <a id="w3c_process_revision" href="https://www.w3.org/2015/Process-20150801/">1 September 2015 <abbr title="World Wide Web Consortium">W3C</abbr> Process Document</a>.</p>
```

Comment:
shouldn't it stay 0901? 0801 returns a 404...

Reply:
Good catch. Fixed